src/aragonjs-wrapper.js

```diff
@@ -0,0 +1,23 @@
+import Aragon from '@aragon/wrapper'
+
+const IPFS_CONF_DEFAULT = {
+  rpc: { host: 'ipfs.infura.io', port: '5001', protocol: 'https' },
```

Comment:
@izqui I think we need to have https on our IPFS node before we can actually use it